src/components/cards/experience-listing-card/ExperienceListingCard.vue

```diff
@@ -0,0 +1,221 @@
+<template>
+  <div class="experience-listing-card">
+    <div class="container">
+      <div class="image">
+        <img :src="image" />
+      </div>
+
+      <div class="body">
+        <p
+          :class="['status', isListed ? 'isListed' : '']"
+          v-if="deviceSize <= deviceSizes.s"
+        >{{ checkListed }}</p>
+        <p class="title">{{ title }}</p>
+        <p class="country" v-if="deviceSize > deviceSizes.s">{{ country }}</p>
+        <div class="actions" v-if="deviceSize <= deviceSizes.s">
+          <div>
+            <a>{{ listingAction }}</a>
+          </div>
+          <template>
+            &nbsp;&bull;&nbsp;
+            <div>
+              <a>View</a>
+            </div>
+          </template>
+        </div>
+      </div>
+
+      <div class="wrapper" v-if="deviceSize > deviceSizes.s">
+        <div class="actions">
+          <div>
+            <a>edit</a>
+          </div>
+          <template v-if="isListed">
+            &nbsp;&bull;&nbsp;
+            <div>
+              <a>preview</a>
+            </div>
+          </template>
+        </div>
+        <div :class="['status', isListed ? 'isListed' : '']">
+          <p>{{ checkListed }}</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { responsiveHandler } from "../../../mixins/responsiveHandler";
```

Comment:
You can omit `;`

Reply:
Also I see that in you all JS you use " instead '

Reply:
Changed